hack/update-codegen.sh

```diff
@@ -54,6 +54,11 @@ bash ${REPO_ROOT_DIR}/hack/generate-groups.sh "deepcopy,client,informer,lister"
   --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
 
 # Depends on generate-groups.sh to install bin/deepcopy-gen
+${PREFIX}/deepcopy-gen \
+  -O zz_generated.deepcopy \
+  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt \
+  -i github.com/tektoncd/pipeline/pkg/apis/pipeline
```

Comment:
Do we also need to change https://github.com/tektoncd/pipeline/blob/1335e0ed75c763fc5822c2c685c63b91f069adee/hack/update-codegen.sh#L48 with the internal version included?

Reply:
I don't think so, another way is to update our `generate-group.sh` to include internal version, similar like https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/vendor/k8s.io/code-generator/generate-internal-groups.sh

Reply:
Got it. I thought L#56 indicating it to be dependent on the previous groups gen.